Assets/Scripts/ProtipManagerScript.cs

```diff
@@ -18,28 +19,19 @@ public class ProtipManagerScript : MonoBehaviour {
 		Invoke ("StartGame", proTipTime);
 		// make this a common shared function somehow
 		int playersActive = 0;
-		int whichSoloPlayer = 0;
```

Comment:
Interesting... where did this logic go? Was it even being used?

Reply:
It wasn't being used! At least that's what the warning said, and I tested it without it successfully.